components/brave_wallet_ui/components/shared/locale-restricted/locale-restricted.test.tsx

```diff
@@ -7,11 +7,15 @@ import { shallow } from 'enzyme'
 
 import { LocaleRestricted } from './locale-restricted'
 
+const currentLocale = Intl.NumberFormat().resolvedOptions().locale
+
+
 describe('<LocaleRestricted />', () => {
   it('renders children when user\'s locale is allowed', () => {
+    expect(currentLocale.length).toBeGreaterThanOrEqual(1)
     const wrapper = shallow((
       <LocaleRestricted
-        allowedLocales={['en-US']}
+        allowedLocales={[currentLocale]}
```

Comment:
I needed to change this as it seems this test doesn't pass if the local machine is not en_US locale. I guess we're just getting lucky @josheleonard?